SabreTools.DatItems/Formats/Disk.cs

```diff
@@ -255,10 +255,14 @@ public Rom ConvertToRom()
                 MD5 = this.MD5,
                 SHA1 = this.SHA1,
 
-                DataArea = new DataArea { Name = this.DiskArea.Name },
```

Comment:
You can just change this to `this.DiskArea?.Name` instead.

Reply:
yep